_components/in-page-navigation/guidance/implementation.md

```diff
@@ -16,12 +16,12 @@ Implementing the in-page navigation component requires a few additions and adjus
   </div>
   ```
 
-{:.usa-content-list }
-- **Root margin.** The root margin property (`data-root-margin`) is a string that allows you to define the observable area. This set of values grows or shrinks the observable area from each side of the root element's bounding box before computing intersections. It can accept values similar to the CSS margin property, and can be percentages. By default, the in-page navigation component is set to `0px 0px 0px 0px`.
+- **Initialization properties.** If needed, you can customize how the component builds during initialization by adding any of the following properties to the `usa-in-page-nav` element. These properties must be set on the element before the component is initialized in order to have an effect.
+- **Custom main content region.** The main content selector property (`data-main-content-selector`) allows you to define which element the component will look in to pull its list of headers. Set the property's value to the class or `id` of your page's main content region (for example, `data-main-content-selector=".main-content"` or `data-main-content-selector="#main-content"`). If the property is not defined, the component will pull headers from the `<main>` element.
+- **Root margin.** The root margin  property (`data-root-margin`) is a string that allows you to define the observable area. This set of values grows or shrinks the observable area from each side of the root element's bounding box before computing intersections. It can accept values similar to the CSS margin property, and can be percentages. By default, the in-page navigation component is set to `0px 0px 0px 0px`.
 - **Threshold.** The threshold property (`data-threshold`) determines how much of the observable section must be present in the observed area before the nav's item is set to current. The default is `1`, which means 100% of the observables must be in the observed section. You can set `data-threshold` to any integer between 0 and 1.
-- **Initialization properties.** The following properties update the component during initialization. These properties must be set before the component is initialized in order to have an effect.
 {% include settings-table-flex.html
   content=page.implementation.initProps
-  cols="flex-2, flex-1, flex-2, flex-1"
+  cols="flex-2, flex-2, flex-1"
   section="initialization properties"
 %}
```

Comment:
Header added in [02a75dc](https://github.com/uswds/uswds-site/pull/2190/commits/02a75dc652cca5f4d2430a015437571eeea78d15). I also adjusted the column width in this commit because adding a header caused the table to span the width of the content area, rather than being constrained to the width of the content list, and I wanted to give a little extra room to the description column.